src/black/strings.py

```diff
@@ -63,6 +63,8 @@ def lines_with_leading_tabs_expanded(s: str) -> List[str]:
             )
         else:
             lines.append(line)
+    if s.endswith("\n"):
```

Comment:
This was necessary so that we don't remove the newline at the end of a module docstring.